src/ai/backend/manager/models/scaling_group.py

```diff
@@ -315,6 +320,45 @@ class ScalingGroup(graphene.ObjectType):
     scheduler_opts = graphene.JSONString()
     use_host_network = graphene.Boolean()
 
+    # Dynamic fields.
+    agent_count = graphene.Int(description="Added in 24.03.0.")
+    agent_total_resource_slots = graphene.JSONString(description="Added in 24.03.0.")
+
+    async def resolve_agent_count(self, info: graphene.ResolveInfo) -> int:
+        from .agent import Agent
+
+        return await Agent.load_count(
+            info.context,
+            scaling_group=self.name,
+        )
+
+    async def resolve_agent_total_resource_slots(
+        self, info: graphene.ResolveInfo
+    ) -> Mapping[str, Any]:
+        from .agent import agents
+
+        graph_ctx = info.context
+        async with graph_ctx.db.begin_readonly() as conn:
+            result = (
+                await conn.execute(
+                    sa.select([agents.c.occupied_slots, agents.c.available_slots]).where(
+                        agents.c.scaling_group == self.name
+                    )
+                )
+            ).fetchall()
```

Comment:
Let's use SQLAlchemy ORM API here rather than Core API.
```suggestion
        async with graph_ctx.db.begin_readonly_session() as db_session:
            query_stmt = sa.select(AgentRow).where(AgentRow.scaling_group == self.name)
            result = await db_session.scalars(query_stmt)
            agent_rows = cast(list[AgentRow], result)
            for row in agent_rows:
                # row.occupied_slots and row.available_slots are ResourceSlot type
            ...
```